rebasehelper/tests/test_utils.py

```diff
@@ -42,6 +44,61 @@
 from rebasehelper.utils import LookasideCacheHelper
 
 
+class TestGitHelper(object):
+
+    @pytest.mark.parametrize('config', [
+        # Get from $XDG_CONFIG_HOME/git/config
+        'global',
+        # Get from $repo_path/.git/config
+        'local',
+        # Get from GIT_CONFIG
+        'env',
+    ])
+    @pytest.mark.usefixtures('workdir')
```

Comment:
You can just pass `workdir` parameter to `test_get_user_and_email` method and you don't have to use `@pytest.mark.usefixtures('workdir')` nor call `os.getcwd()`.